examples/config/profile_script_geo.yaml

```diff
@@ -0,0 +1,26 @@
+---
+type: profile
+kind: script
+configuration:
+  # Profile name - a unique name for the profile
+  # Consider using a unique namespace of either an organisation or project
+  profile-name : thingful/device-test-geo
+  # Profile description
+  profile-description : blah blah blah
+  # Semanatic version for the profile
+  profile-version: 1.0.0-beta
+  # Name of the function to call
+  script-main: xxx
+  # Runtime to use - valid value 'javascript'
+  script-runtime: javascript
+  # Format of the message received.
+  # One of 'json', 'csv', 'xml', 'raw'
+  script-input: json
+  # Function to run - written in Javascript
+  # Can log via console.log
+  # Must return an object
+  script-contents: >
+      function xxx ( input )
+      {
+        return Number(geolocation.Coords.Latitude).toString() + ',' +  Number(geolocation.Coords.Longitude).toString()
```

Comment:
Minor - Can the example be cleaned up to not include the casting as it should be a number already?